### ISSUE TITLE
Livrable 1

### ISSUE THREAD

**Issue by Pmnin** [Pmnin/inf3135-hiv2016-tp2#1]
## Tâches pour le Livrable 1

- [ ] Conception d'un affichage d'une page d'accueil avec SDL
- [ ] Conception d'un README.md de base avec l'énumération des tâches détaillées
- [ ] Conception d'un Makefile permettant de créer un exécutable pour les plateformes MacOS & Linux

**Comment by ablondin** [comment]
Désolé pour le lien non fonctionnel, j’ai fait du ménage dans mes dépôts.

Ça devrait maintenant fonctionner.

Le 2016-02-23 à 15:53, P Michel <notifications@github.com<mailto:notifications@github.com>> a écrit :

Tâches pour le Livrable 1

  *    Git Fork des composantes jeu du projet de @ablondin<https://github.com/ablondin> Lien mort à l'instant du 02/23/16<https://bitbucket.org/ablondin/inf3135-hiv2016-tp2>
  *    Conception d'un affichage d'une page d'accueil avec SDL
  *    Conception d'un README.md de base avec l'énumération des tâches détaillées
  *    Conception d'un Makefile permettant de créer un exécutable pour les plateformes MacOS & Linux

—
Reply to this email directly or view it on GitHub<https://github.com/Pmnin/inf3135-hiv2016-tp2/issues/1>.

Alexandre Blondin Massé
Professeur adjoint/Associate professor
Département d’informatique/Computer Science Department
Université du Québec à Montréal
Case postale 8888, Succursale Centre-ville
Montréal (Québec) H3C 3P8
http://thales.math.uqam.ca/~blondin
Tél./Tel.: 514-987-3000 ext. 5516

**Comment by Pmnin** [comment]
@ablondin Avez-vous mis votre dépôt (https://bitbucket.org/ablondin/inf3135-hiv2016-tp2) Bitbucket public?

**Comment by ablondin** [comment]
Oui, le dépôt est public, mais c’est le lien qui a changé. Tu dois prendre
le nouveau lien, qui est :
https://bitbucket.org/ablondin-projects/inf3135-hiv2016-tp2

Le 2016-02-23 à 16:47, P Michel <notifications@github.com<mailto:notifications@github.com>> a écrit :


@ablondin<https://github.com/ablondin> Avez-vous mis votre dépôt (https://bitbucket.org/ablondin/inf3135-hiv2016-tp2) Bitbucket public?

—
Reply to this email directly or view it on GitHub<https://github.com/Pmnin/inf3135-hiv2016-tp2/issues/1#issuecomment-187928818>.

Alexandre Blondin Massé
Professeur adjoint/Associate professor
Département d’informatique/Computer Science Department
Université du Québec à Montréal
Case postale 8888, Succursale Centre-ville
Montréal (Québec) H3C 3P8
http://thales.math.uqam.ca/~blondin
Tél./Tel.: 514-987-3000 ext. 5516

**Comment by Pmnin** [comment]
D'accord, ça fonctionne.
Merci!

**Comment by ablondin** [comment]
Tu ne devrais pas faire un dépôt public, il y a des risques de plagiat... Sur Bitbucket, c'est dépôts privés illimités. Sur Github, tu peux en avoir 5 gratuitement si tu appliques pour une licence académique : https://education.github.com/.

**Comment by Pmnin** [comment]
D'accord, je vais changer vers Bitbucket.
Je ne peux pas fork+pull request sur Github est surtout la raison principale pour le transfert.

**Comment by ablondin** [comment]
Mais tu n’as pas besoin de faire de pull request. Tu fais un fork de mon
projet, mais je ne le fusionnerai pas !

En fait, tu as déjà fait pas mal de trucs sur Github, je ne veux pas te faire
tout recommencer, mais faire un projet public est risqué, c’est ce que je
voulais dire.

De plus, le fork n’est pas obligatoire à proprement parler. Tu peux simplement
cloner le projet et ensuite le mettre sur Github.

A+

Le 2016-02-23 à 18:01, P Michel <notifications@github.com<mailto:notifications@github.com>> a écrit :


D'accord, je vais changer vers Bitbucket.
Je ne peux pas fork+pull request sur Github est surtout la raison principale pour le transfert.

—
Reply to this email directly or view it on GitHub<https://github.com/Pmnin/inf3135-hiv2016-tp2/issues/1#issuecomment-187954718>.

Alexandre Blondin Massé
Professeur adjoint/Associate professor
Département d’informatique/Computer Science Department
Université du Québec à Montréal
Case postale 8888, Succursale Centre-ville
Montréal (Québec) H3C 3P8
http://thales.math.uqam.ca/~blondin
Tél./Tel.: 514-987-3000 ext. 5516

**Comment by Pmnin** [comment]
Pas de problèmes, ça pratique avec [Bitbucket](https://bitbucket.org/pmichel/inf3135-hiv2016-tp2)
Mais vu que j'ai pas Jira, ça bloque un peu pour le suivi et organisation du projet avec des issues et des milestones.

Bonne soirée
